docs/conf.py

```diff
@@ -11,14 +11,21 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+from __future__ import annotations
+
 import importlib
 import os
+import subprocess
 import sys
-from typing import Optional
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from sphinx.application import Sphinx
 
 import tmt.utils
 
-_POSSIBLE_THEMES: list[tuple[Optional[str], str]] = [
+_POSSIBLE_THEMES: list[tuple[str | None, str]] = [
```

Comment:
Isn't this syntax Python 3.10+ only?

Reply:
Apparently we can use these as long as we include `from __future__ import annotations`

Reply:
That being true, I'd prefer not to introduce this syntax in unrelated patches, in one file of many. Very recently, we moved from `List`, `Dict`, and `Tuple` to their native counterparts - there was a commit focused on this transition, reviewers knew this was the purpose of the PR, and so on. I'd recommend not changing these lines as the change is not related to the goal of the PR.